torchvision/transforms/v2/functional/_meta.py

```diff
@@ -345,30 +348,22 @@ def convert_dtype_image_tensor(image: torch.Tensor, dtype: torch.dtype = torch.f
             return image.to(dtype).bitwise_left_shift_(num_value_bits_output - num_value_bits_input)
 
 
-# We changed the name to align it with the new naming scheme. Still, `convert_image_dtype` is
-# prevalent and well understood. Thus, we just alias it without deprecating the old name.
-convert_image_dtype = convert_dtype_image_tensor
-
+def to_dtype_video(video: torch.Tensor, dtype: torch.dtype = torch.float, scale: bool = False) -> torch.Tensor:
+    return to_dtype_image_tensor(video, dtype, scale=scale)
 
-def convert_dtype_video(video: torch.Tensor, dtype: torch.dtype = torch.float) -> torch.Tensor:
-    return convert_dtype_image_tensor(video, dtype)
 
-
-def convert_dtype(
-    inpt: Union[datapoints._ImageTypeJIT, datapoints._VideoTypeJIT], dtype: torch.dtype = torch.float
-) -> torch.Tensor:
+def to_dtype(inpt: datapoints._InputTypeJIT, dtype: torch.dtype = torch.float, scale: bool = False) -> torch.Tensor:
     if not torch.jit.is_scripting():
-        _log_api_usage_once(convert_dtype)
+        _log_api_usage_once(to_dtype)
 
     if torch.jit.is_scripting() or is_simple_tensor(inpt):
-        return convert_dtype_image_tensor(inpt, dtype)
+        return to_dtype_image_tensor(inpt, dtype, scale=scale)
     elif isinstance(inpt, datapoints.Image):
-        output = convert_dtype_image_tensor(inpt.as_subclass(torch.Tensor), dtype)
+        output = to_dtype_image_tensor(inpt.as_subclass(torch.Tensor), dtype, scale=scale)
         return datapoints.Image.wrap_like(inpt, output)
     elif isinstance(inpt, datapoints.Video):
-        output = convert_dtype_video(inpt.as_subclass(torch.Tensor), dtype)
+        output = to_dtype_video(inpt.as_subclass(torch.Tensor), dtype, scale=scale)
         return datapoints.Video.wrap_like(inpt, output)
-    else:
-        raise TypeError(
-            f"Input can either be a plain tensor or an `Image` or `Video` datapoint, " f"but got {type(inpt)} instead."
-        )
+    elif not isinstance(inpt, torch.Tensor):
+        raise TypeError(f"Expected a tensor or a datapoint, but got {type(inpt)} instead.")
+    return inpt.to(dtype)
```

Comment:
```suggestion
    elif isinstance(inpt, datapoints._datapoint.Datapoint):
        return inpt.to(dtype)
    else:
        raise TypeError(
            f"Input can either be a plain tensor or an `Image` or `Video` datapoint,
            " f"but got {type(inpt)} instead.
        )
```

I prefer to sticking to regular dispatcher "layout" here:

1. It took me a few glances to actually see that we indeed have a `return inpt.to(dtype)` there. I think this comes from the fact that all other dispatchers don't do this. This is likely only going to happen to us maintainers since the code is clear, but still.
2. Right now, the dispatcher supports all tensors. Although I agree that `to_dtype` is general enough to allow this, all other dispatchers only support plain tensors and datapoints. And since for all other cases this is just a fancy wrapper around `t.to(dtype)`, I don't see a point of supporting that here.